marklogic-data-hub/src/test/java/com/marklogic/hub/security/DataHubSecurityAdminTest.java

```diff
@@ -95,4 +98,114 @@ public void task5CannotCreateCustomRoleInheritingExistingRole() {
             new Role(adminUserApi, roleName).delete();
         }
     }
+
```

Comment:
All these tests would fail in versions < 10.0-3 . I think these tests have to be disabled based on HubTestBase. isVersionCompatibleWithGranularPrivilege() like PR #3391 .

Reply:
Good catch, I copied that method over from the 3504 PR.